src/docs/product/accounts/quotas/index.mdx

```diff
@@ -7,317 +7,116 @@ redirect_from:
 
-Spike protection is an organization-level setting, so once it's triggered, it affects all the projects in the organization, regardless of which project triggered the spike.
+### 2. Adjusting Your Quota {#increasing-quotas}
 
-Spike protection is enabled for your organization by default, and when it's enabled, Sentry continually monitors for spikes. If you want to disable it, you can do so in **Settings > Subscription**.
+Events and attachments that exceed your quota are not accepted, so you may want to increase your quota. Conversely, you might want to decrease your quota or adjust your reserved and on-demand quotas to better control your spending. Learn about adjusting your quota in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#2-adjusting-quotas), [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#1-adjusting-quotas), and [Manage Your Attachments Quota](/product/accounts/quotas/manage-attachments-quota/#1-adjusting-quotas).
 
-### How Does Spike Protection Work?
+### 3. Rate Limits {#limiting-events}
 
-We use your historical error event volume to implement a dynamic rate limit, and then discard error events when you hit its threshold. When spike protection is activated, we limit the number of error events accepted in any minute to:
+You can add limits for error events on a per-project basis in **[Project] > Settings > Client Keys (DSN)**. If the event rate limit for a project has been exceeded, and your subscription allows, the event won't be counted. You can also rate limt attachments on an organization level in **Settings > Security & Privacy**. Learn more in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) and [Manage Your Attachments Quota](/product/accounts/quotas/manage-attachments-quota/#2-rate-limiting).
 
-```
-maximum(20, 6 x average error events per minute over the last 24 hours)
-```
+### 4. Event Repetition
 
-<Note>
+In some cases, repeated events can count against your quota, so it's important to know how event repetition is treated in Sentry. Learn more in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#5-applying-workflows).
 
-The 24-hour window ends at the beginning of the current hour, not at the current minute.
+### 5. Inbound Filters {#inbound-data-filters}
 
-</Note>
+If an inbound filter is set for a type of error, transaction, or attachment, and your subscription allows, it won't be counted. You can set these in **[Project] > Settings > Inbound Filters**. Learn more in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#3-inbound-data-filters), [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#2-inbound-data-filters), and [Manage Your Attachments Quota](/product/accounts/quotas/manage-attachments-quota/#3-inbound-data-filters).
```

Comment:
Consider "If an inbound filter is applied to an error, ..." Inbound filters can't be set for specific transactions or attachments but some inbound filters do apply to all data categories.

Reply:
> Consider "If an inbound filter is applied to an error, ..." Inbound filters can't be set for specific transactions or attachments but some inbound filters do apply to all data categories.

Do you know which ones apply in which cases? I just updated that content recently from saying inbound filters don't apply to transactions and attachments at all to saying it does and I wasn't given any indication that it was specific.

Reply:
IP filters should filter all event types. As does filtering by release I believe. You might want to verify that with the Ingest team though.